samples/bot-daily-task-reminder/csharp/README.md

```diff
@@ -0,0 +1,112 @@
+﻿﻿﻿
+## Bot daily task reminder
+
+This sample shows feature where user can schedule a recurring task and get reminder on the scheduled time.
+
+![Schedule task ](BotDailyTaskReminder/Images/ScheduleTaskCard.png)
+
+![Task Details](BotDailyTaskReminder/Images/ScheduleTask.png)
+
+![Task reminder](BotDailyTaskReminder/Images/TaskReminder.png)
+
+## Prerequisites
+
+- [.NET Core SDK](https://dotnet.microsoft.com/download) version 3.1
+
+  determine dotnet version
+  ```bash
+  dotnet --version
+  ```
+- [Ngrok](https://ngrok.com/download) (For local environment testing) Latest (any other tunneling software can also be used)
+  
+  run ngrok locally
+  ```bash
+  ngrok http -host-header=localhost 3978
+  ```
+- [Teams](https://teams.microsoft.com) Microsoft Teams is installed and you have an account
+
+## To try this sample
+
+1) Setup for Bot
+   In Azure portal, create a [Azure Bot resource](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-authentication?view=azure-bot-service-4.0&tabs=csharp%2Caadv2).
+   
+2) Clone the repository
+   ```bash
+   git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+   ```
+   
+3) In a terminal, navigate to `samples/bot-task-reminder/csharp`
```

Comment:
Which terminal??
This is not a nodejs sample.
So we need to mention the proper steps for opening the project in Visual studio etc.